src/components/structured-list/_structured-list.scss

```
@@ -174,3 +174,157 @@
     @include typescale('zeta');
   }
 }
+
+@mixin structured-list--x {
+  .#{$prefix}--structured-list--selection .#{$prefix}--structured-list-td,
+  .#{$prefix}--structured-list--selection .#{$prefix}--structured-list-th {
+    @include padding--data-structured-list--x;
+  }
+
+  .#{$prefix}--structured-list-input {
+    display: none;
+  }
+
+  .#{$prefix}--structured-list {
+    @include reset;
+    @include font-family;
+    display: table;
+    overflow-x: auto;
+    overflow-y: hidden;
+    width: 100%;
+    min-width: 500px;
+    border-collapse: collapse;
+    border-spacing: 0;
+    margin-bottom: 5rem;
+    background-color: transparent;
+
+    &.#{$prefix}--structured-list--condensed .#{$prefix}--structured-list-td,
+    &.#{$prefix}--structured-list--condensed .#{$prefix}--structured-list-th {
+      @include padding-td--condensed--x;
+    }
+  }
+
+  .#{$prefix}--structured-list-row {
+    display: table-row;
+    border-bottom: 1px solid $ui-03;
+    transition: $transition--base $carbon--standard-easing;
+
+    .#{$prefix}--structured-list--selection &:hover:not(.#{$prefix}--structured-list-row--header-row) {
+      background-color: $ibm-colors__gray--10;
+      cursor: pointer;
+    }
+
+    &.#{$prefix}--structured-list-row--selected {
+      background-color: $ibm-colors__gray--20;
+    }
+
+    &.#{$prefix}--structured-list-row--header-row {
+      border-bottom: 1px solid $ibm-colors__gray--20;
+      cursor: inherit;
+    }
+
+    &:focus:not(.#{$prefix}--structured-list-row--header-row) {
+      outline: 2px solid $ibm-colors__blue--60;
```

Should be able to use the focus mixin here, looks like the spec has it on the inside anyway. 
```suggestion
      @include focus-outline('outline');
```